python/lsst/ts/idl/enums/__init__.py

```diff
@@ -18,34 +18,36 @@
 #
 # You should have received a copy of the GNU General Public License
 
-from . import ATCamera
-from . import ATDomeTrajectory
-from . import ATDome
-from . import ATHexapod
-from . import ATMCS
-from . import ATMonochromator
-from . import ATPneumatics
-from . import ATPtg
-from . import ATSpectrograph
-from . import ATThermoelectricCooler
-from . import ATWhiteLight
-from . import Electrometer
-from . import FiberSpectrograph
-from . import Guider
-from . import LaserTracker
-from . import LinearStage
-from . import MTAOS
-from . import MTDome
-from . import MTDomeTrajectory
-from . import MTHexapod
-from . import MTM1M3
-from . import MTM2
-from . import MTMount
-from . import MTPtg
-from . import MTRotator
-from . import PMD
-from . import Scheduler
-from . import Script
-from . import ScriptQueue
-from . import TunableLaser
-from . import Watcher
+from . import (
```

Comment:
I'm a bit nervous allowing isort to run on `__init__.py` files, for fear of breakage. But in this case it's probably fine -- circular dependencies aren't likely.